### PR TITLE
ML Docker

### PR DESCRIPTION
**Pull request by acheshkov**

### CODE REVIEW

**Comment by acheshkov** [comment]
@0crat out

**Comment by 0crat** [comment]
> @0crat out ([here](https://github.com/yegor256/aibolit/issues/309#issuecomment-628824689))

@acheshkov Job `gh:yegor256/aibolit#309` is not assigned, can't get performer

**Comment by 0crat** [comment]
This pull request [#309](https://github.com/yegor256/aibolit/issues/309) is assigned to @acheshkov[/z](https://www.0crat.com/u/acheshkov), here is [why](https://www.0crat.com/footprint/CT5UB7XAP/5c3dda32-ce57-432b-b1c6-2bef4a38250a); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @acheshkov[/z](https://www.0crat.com/u/acheshkov) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; we should be aware that @acheshkov[/z](https://www.0crat.com/u/acheshkov) is on vacation; this ticket may be delayed; there will be no monetary reward for this job

**Comment by 0crat** [comment]
> @0crat out ([here](https://github.com/yegor256/aibolit/issues/309#issuecomment-628824689))

@acheshkov @acheshkov[/z](https://www.0crat.com/u/acheshkov) resigned from [#309](https://github.com/yegor256/aibolit/issues/309), since the job is not in scope anymore

**Comment by 0crat** [comment]
> @0crat out ([here](https://github.com/yegor256/aibolit/issues/309#issuecomment-628824689))

@acheshkov Pull request [#309](https://github.com/yegor256/aibolit/issues/309) was not merged, no payment for `ARC`, see [§28](http://www.zerocracy.com/policy.html#28)

File: scripts/02-filter-and-move.py [hunk 1]
```diff
@@ -37,11 +38,17 @@
 parser.add_argument(
     '--dir',
     help='dir for Java files search',
-    required=True)
-parser.add_argument('--max_classes', type=int, required=False, default=None)
+    required=True
```
Comment (lyriccoder) [inline scripts/02-filter-and-move.py:41]: set required=False

**Comment by acheshkov** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job `gh:yegor256/aibolit#309` is not assigned, can't get performer

**Pull request merged**